elements/pl-graph/pl-graph.py

```diff
@@ -47,10 +47,14 @@ def graphviz_from_adj_matrix(
     )
     input_param_name = pl.get_string_attrib(element, "params-name", input_param_matrix)
 
+    # Exception to make typechecker happy.
+    if input_param_name is None:
+        raise ValueError('"params-name" is a required attribute.')
+
```

Comment:
Is this still necessary? It looks like you defined overloads for the case where a default is provided, so I'd expect the type checker to know that `input_param_name` is not in fact `None` here.

Reply:
Unfortunately, it does not. The reason is because this uses a passthrough, and the old default value of `None` is still getting used. I think this wasn't here originally because the `None` check was happening outside of this function.

Reply:
Ah, ok, I see that now when looking at the broader context here.